### PR TITLE
Redesign of external website 2023 - Index page

### PR DESCRIPTION
**Pull request by Ellen-Wittingen**
See the files on NextCloud for what the next external website should look like.

### CODE REVIEW

**Comment by DrumsnChocolate** [comment]
I have not looked at the code yet but this is looking pretty good visually!

**Comment by DrumsnChocolate** [comment]
there are some aspect ratio's where the top header is a bit less aesthetically pleasing:
image

**Comment by Ellen-Wittingen** [comment]
> there are some aspect ratio's where the top header is a bit less aesthetically pleasing:
> image
> 

Try hard refreshing, in local mode, it grabs your browsers default font (like it did here), the actual font is smaller and fits better

**Review by wilco375 — CHANGES_REQUESTED** [review]
Some remarks:
- Ads are no longer stacked vertically on top of each other
- Maybe add a Youtube icon next to FB and IG?
- Did not test if this is caused by this PR but the background of filter boxes has changed color:
image
- On Chrome, navbar on public side shows an unscrollable scrollbar, and there's no margin between the flag and edge of the page
image
- On mobile when expending about us, there is a lot of padding below the item:
image
- When clicking on an activity, nothing happens (console prints `this.activity is undefined`)
- Not sure if related to this PR but if you view an individual image, there's no padding between the image and responses (when logged in, the uploader's name is in between these two elements)
- NL flag has round corners, EN flag does not
- On mobile, padding is quite big in this section. Decreasing it will give a bit more room to the activity and photo album elements. Same goes for videos section.
image

**Comment by Ellen-Wittingen** [comment]
My replies to Wilco's comments (in order)
- This was intentional, I thought it looked better if all advertisements were uniform, especially now that they have a white square background. They are still stacked when logged in.
- If we want to use the youtube logo to link to the Alpha youtube channel, we need to fill in YouTube's request form (this was not the case for facebook and instagram cuz meta has different rules than google). I could fill it in, but Im not sure its worth the trouble.
- Fixed
- Fixed (For the flag, I forgot that chrome and firefox have different scrollbar behaviours, so annoying)
- Fixed. But could only be done by getting rid of being able to toggle it dropping down or not, as that needs a fixed height (so cannot be dependent on the height of its children) for the dropdown animation. Now its permanently open.
- It works fine for me in Firefox and Chromium, so could not reproduce
- Fixed (was not caused by this PR, but was fixable with one line)
- Fixed (images did not have the same height-width ratiios, so scaled the british flag to be higher)
- Fixed

**Comment by DrumsnChocolate** [comment]
I don't have time to review the whole PR right now, but I just wanted to chime in on this discussion:
> https://arxiv-org.ezproxy2.utwente.nl/abs/2111.11432

The layout of the ads is tied to how much we ask for each ad. We created stacked ads to be cheaper because they take less space and are therefore visually less intrusive/valuable. Changes to the layout of ads influence the price we should be asking for ads; it's weird to give the same amount of space to an ad that pays less.

**Review by DrumsnChocolate — APPROVED** [review]
I'm not entirely enthusiastic about the new look of the adverts, they are more intrusive than before due to the increased contrast with the background.
For comparison:
new
image
old
image

Not sure what can be done about this with the current palette

File: app/components/menu-sidebar.js [hunk 1]
```diff
@@ -21,4 +24,18 @@ export default class MenuSidebar extends Component {
...
+    this.store.findAll('static-page').then((pages) => {
+      // make key-value pairs for all found static pages that do not appear by itself
+      let staticPages = {};
+      pages.forEach((page) => {
+        if (!['word-lid', 'sponsoring'].includes(page.id)) {
+          staticPages[page.id] = page.title;
+        }
+      });
+      this.staticPages = staticPages;
+    });
```
[the remark this answers is not in the document]
Comment (Ellen-Wittingen) [reply app/components/menu-sidebar.js:39]: I also do not know where to extract it to. In the future we will not use any static page in the public part of the website anymore, so then this code will be removed anyways. So I suggest leaving it like it is now.

**Pull request merged**